MovieGallery/Pages/OurTeam.cshtml

```diff
@@ -11,7 +11,7 @@
     </div>
     <div class="row no-gutters">
         <div class="col-sm-5">
-            <img src="\images\Rutuja.png" class="card-img-top h-75" alt="Rutuja Sawant">
+            <img src="\images\Rutuja.png" class="card-img-top h-75" alt="Rutuja Sawant" />
```

Comment:
Added end tag in the img src tag because empty elements must always be closed
https://www.w3schools.com/html/html_xhtml.asp